traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -544,6 +544,10 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.3, http.MethodPost, `deliveryservices/?(\.json)?$`, deliveryservice.CreateV13, auth.PrivLevelOperations, Authenticated, nil, 1705681904, noPerlBypass},
 		{1.1, http.MethodPost, `deliveryservices/?(\.json)?$`, deliveryservice.CreateV12, auth.PrivLevelOperations, Authenticated, nil, 652813412, noPerlBypass},
 
+		{1.4, http.MethodPut, `deliveryservices/{id}/safe/?(\.json)?$`, deliveryservice.UpdateSafeV14, auth.PrivLevelOperations, Authenticated, nil, 547210934, noPerlBypass},
+		{1.3, http.MethodPut, `deliveryservices/{id}/safe/?(\.json)?$`, deliveryservice.UpdateSafeV13, auth.PrivLevelOperations, Authenticated, nil, 547210933, noPerlBypass},
+		{1.1, http.MethodPut, `deliveryservices/{id}/safe/?(\.json)?$`, deliveryservice.UpdateSafeV12, auth.PrivLevelOperations, Authenticated, nil, 547210931, noPerlBypass},
```

Comment:
Would you be opposed to keeping this API "unversioned" or pinned to one particular minor version? It would be nice to keep the minor version overhead down where possible, and this API is designed to be safe enough to not need minor versioning promises (since it only cares about those 4 "safe" fields). If we added another "safe" field, we could add a new minor version for that instead?